torchtune/training/checkpointing/_checkpointer.py

```diff
@@ -709,6 +718,8 @@ def save_checkpoint(
                     dim=self._config["hidden_size"],
                     head_dim=self._config.get("head_dim", None),
                 )
+            elif self._model_type == ModelType.FLUX:
+                pass  # the torchtune Flux model state dict is identical to the huggingface one
```

Comment:
Not a problem for this PR, but in this big if/elif/.../elif/else I think this (no weight conversion) should actually be the else, not Llama2 (for whatever reason)